### ISSUE TITLE
UnboundLocalError local variable 'holdings_id'

### ISSUE THREAD

**Issue by Daltorch** [wtee/alma_enumerator#1]
I'm getting the following:

  File "ae_gui", line 30, in update_it
    update(settings.mms_id, settings.input_file, settings.error_file, settings.api_key, settings.base_url)
  File "C:\AlmaEnumerator\alma_enumerator-master\AE.py", line 524, in update
    items_to_update = get_info_from_csv(input_file)
  File "C:\AlmaEnumerator\alma_enumerator-master\AE.py", line 477, in get_info_from_csv
    item_info[holdings_id].append(dict(zip(keys, info)))
UnboundLocalError: local variable 'holdings_id' referenced before assignment

I've tried a couple different things, but just get various errors.

**Comment by nausicaa-rose** [comment]
Thanks for reaching out about this bug! Would you be able to provide the input file you're using to update your records?

My initial guess is that there's a mismatch between the input CSV's format and what the `get_info_from_csv` function is expecting.

**Comment by Daltorch** [comment]
Sure. Here's the output file created after fetching the data
[output.txt](https://github.com/wtee/alma_enumerator/files/3473609/output.txt)
Thanks for taking a look at this.

**Comment by nausicaa-rose** [comment]
OK. The problem was the code was overly precise about identifying the holdings ID in the CSV. When I initially developed Alma Enumerator, all the holdings MMS IDs were 16 digits in my collection, so I matched for that. I've updated it so it should recognize any MMS ID of a valid length (8-19 digits.)

If you replace your AE.py file with the new one, that should take care of your problem. Let me know if it doesn't or if you encounter other issues.
